packages/mds-policy-service/repository/index.ts

```diff
@@ -250,7 +243,7 @@ class PolicyReadWriteRepository extends ReadWriteRepository {
   public publishPolicy = async (policy_id: UUID, publish_date = now()) => {
     try {
       if (await this.isPolicyPublished(policy_id)) {
-        throw new AlreadyPublishedError('Cannot re-publish existing policy')
+        throw new ConflictError('Cannot re-publish existing policy')
```

Comment:
Above 10 lines below, ought to delete all of the commented out codez

Reply:
Thanks!